src/clj/witan/send/params.clj

```diff
@@ -104,23 +107,24 @@
         joiner-calendar-years (keys joiners)
         population (calculate-population-per-calendar-year population)
         academic-years (->> (map first valid-states) distinct sort)
-        n (count joiner-calendar-years)
-        params (reduce
-                (fn [coll ay]
-                  (reduce
-                   (fn [coll cy]
-                     (let [j (get-in joiners [cy ay])
-                           p (get-in population [cy ay])]
-                       (if j
-                         (-> coll
-                             (update-in [ay :alpha] m/some+ (/ j n))
-                             (update-in [ay :beta] m/some+ (/ (- p j) n)))
-                         coll)))
-                   coll
-                   joiner-calendar-years))
-                {}
-                academic-years)]
-    (continue-for-latter-ays params academic-years)))
+        n (count joiner-calendar-years)]
+    (reduce
+     (fn [coll ay]
+       (reduce
+        (fn [coll cy]
+          (let [j (get-in joiners [cy ay])
+                p (get-in population [cy ay])]
+            (if j
+              (-> coll
+                  (update-in [ay :alpha] m/some+ (/ j n))
+                  (update-in [ay :beta] m/some+ (/ (- p j) n)))
+              (-> coll
+                  (update-in [ay :alpha] m/some+ 0.001) ; default prior may need tweaking
```

Comment:
It looks like from your comment that you are unsure of this number, but I am presuming at this point it is just a nice low number as default?

Reply:
Yep.